src/v3/src/components/Form/Form.tsx

```diff
@@ -50,7 +57,7 @@ const Form: FunctionComponent<{
         .reduce((acc: Record<string, Partial<IdxMessage & { name?: string }>[]>, curr) => {
           const name = curr[0];
           const elementSchema = curr[1] as DataSchema;
-          if (typeof elementSchema.validate === 'function') {
+          if (fieldsToValidate.includes(name) && typeof elementSchema.validate === 'function') {
```

Comment:
This seems like we are narrowing the number of times where we call the validation function, is that correct?

Reply:
it's introduced to support form with stepper schema, so when step switches, it updates the fields that need to be validate in that specific step. See security question enrollment schema for example.